ghcide/bench/lib/Experiments.hs

```diff
@@ -563,6 +563,9 @@ setup = do
                 writeFile
                     (path </> "cabal.project.local")
                     ""
+                -- Fix newlines, required for GHC 9.2 and lsp-types
```

Comment:
Why is this required? Does this mean HLS has some troubles when working on `lsp-types`?

Reply:
Honestly, I have no idea. The issue does not repro with ghc 8.10.7, which makes me think it's not an HLS bug. But it doesn't repro in `cabal repl` with ghc 9.2 either, so 🤷‍♂️🤷‍♂️🤷‍♂️